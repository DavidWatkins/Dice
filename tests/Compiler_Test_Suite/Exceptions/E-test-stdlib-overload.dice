class test {

	public void print(){

	}

  public void main(char[][] args) {
     
  }
}