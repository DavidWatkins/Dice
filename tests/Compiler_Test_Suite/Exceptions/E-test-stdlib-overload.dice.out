Cannot use name print because it is reserved
